### ISSUE TITLE
针对ControlNet1.1

### ISSUE THREAD

**Issue by TownSolomon** [ZhUyU1997/open-pose-editor#59]
ControlNet1.1的新版OpenPose模型可以直接识别手部形状和面部表情，实际效果比深度图好一些，可以考虑针对更新
![openpose_1](https://user-images.githubusercontent.com/101002348/231967745-a6934bb3-6160-4ea2-86d3-066ceaf67398.png)

**Comment by ZhUyU1997** [comment]
目前没有支持webui主线？稍后会考虑添加

**Comment by ostap667inbox** [comment]
No, ControlNet 1.0 also supports this type of OpenPose. Everyone is looking forward to adding such a feature to the Open Pose Editor.
![chrome_7MUEPkiREf](https://user-images.githubusercontent.com/123593542/232231765-708a59d8-5b7d-493b-aade-3c11ee6f9a14.png)

**Comment by DanielSHKao** [comment]
Same issue here. Would you still consider adding the output for openpose_full model, please?
